tools/pythonpkg/src/vector_conversion.cpp

```diff
@@ -313,6 +313,23 @@ void VectorConversion::BindPandas(py::handle df, vector<PandasColumnBindData> &b
 	if (py::len(df_columns) == 0 || py::len(df_types) == 0 || py::len(df_columns) != py::len(df_types)) {
 		throw std::runtime_error("Need a DataFrame with at least one column");
 	}
+
+	// check if names in pandas dataframe are unique
+	unordered_map<string, idx_t> pandas_column_names_map;
+	py::array column_attributes = df.attr("columns").attr("values");
+	for (idx_t col_idx = 0; col_idx < py::len(df_columns); col_idx++) {
+		pandas_column_names_map[py::str(df_columns[col_idx])]++;
```

Comment:
The performance here is unlikely to matter, but it seems somewhat wasteful to call `py::str(df_columns[col_idx])` so many times. Could we call it once and extract it into a `string` variable?